Lab2_Code.m

```diff
@@ -0,0 +1,204 @@
+% with itself 8 times. Afterwards, I must integrate my distribution from
+% negative infinity to the signal to find the associated sigma. 
+pconv = multiconv(pbg,8);
+psum = 0;
+for c = 1:128
+   psum = psum+pconv(c); 
+end
+psigma = norminv(psum);
+disp(psigma);
+
+%% Problem 2
+
+%% Section 2A
+% Shows how Rayleigh distribution changes as we average over more observing
+% intevals. Similar to the previous problem, we notice how the mean doesn't
+% shift and the amplitude at the mean decreases. Also, the shape of the
+% distribution approximates a normal distribution when the sample size
+% becomes large enough according to the Central Limit Theorem. 
+rdist = makedist("Rayleigh");
+rbg = pdf(rdist,0:3);
+
+% Average over 15 observing intervals
+rconv = multiconv(rbg,15);
+rx = 0:length(rconv)-1;
+plot(rx/15,rconv); hold on;
+
+% Average over 20 observing intervals
+rconv = multiconv(rbg,20);
+rx = 0:length(rconv)-1;
+plot(rx/20,rconv); hold on;
+
+% Average over 30 observing intervals
+rconv = multiconv(rbg,30);
+rx = 0:length(rconv)-1;
+plot(rx/30,rconv); hold on;
+
+% Average over 40 observing intervals
+rconv = multiconv(rbg,40);
+rx = 0:length(rconv)-1;
+plot(rx/40,rconv); hold on;
+
+title("Average Probability Distribution After N Intervals"); 
+xlabel("Observations"); ylabel("Probability"); 
+legend("N = 10","N = 20","N = 30","N = 40"); hold off;
+
+%% Section 2B
+% According to the Central Limit Theorem, the distribution should
+% approximate a normal distribution when the sample size gets large. As the
+% average over more observing intervals increased, the Rayleigh
+% distribution changed. Specifically, the width of the distribution became
+% thinner, the amplitude at the mean decreased, and the shape became
+% smoother. However, the mean did not shift. The Rayleigh distribution
+% approached a Gaussian distribution after approximately 30 intervals. 
+
+%% Problem 3
+% For this section, I selected the width of the background Gaussian
+% distribution as X = 5 and the signal of strength as Y = 8.6. 
+
+%% Section 3A (Version 1)
+% Here, we make a Gaussian distribution with mu = 0 and sigma = 5.
+% Afterwards, we calculate the probability of an 8.6 signal using the cdf()
+% function. We put this value into the norminv() function calculate the
+% sigma. From this, we get a sigma of 1.72 which means we cannot claim a
+% discovery since that's associated with 5-sigma. 
+gdist = makedist("Normal",'mu',0,'sigma',5);
+gprob = cdf(gdist,8.6);
+gsigma = norminv(gprob);
+disp(gsigma);
+
+%% Section 3B (Version 2)
+% The statistical question I want to ask is...
+gdist = makedist("Normal",'mu',0,'sigma',5);
+gpdf = pdf(gdist,-30:30);
+gprobf = gpdf*10000;
+gbg = plot(-30:30,gprobf);
+title("Probability Distribution of Background");
+xlabel("Observations"); ylabel("Probability");
+
+% The probability we get at least one signal-like event from the background
+% is the probability we get from one of these pixels. If we assume the
```

Comment:
I think you dropped the the 10k. Also also if we multiply by 10k we don't have a probability anymore since the distribution is not normalized anymore it becomes and distribution of the expected counts at each value